src/App/App.css

```diff
@@ -7,12 +7,12 @@
 }
 
 .nav-main {
-  height: 20%;
+  height: auto;
```

Comment:
The layout of the app looked really good on my screen - I didn't see any issues. The only thing I saw was when I made the screen smaller, the margin on the left side of the page went away. Maybe adding some padding to the card container would help! 

As for whether to wrap things in divs -- you shouldn't have to do that since each component already has a wrapper on it (since React components can only return a single HTML element). You could target all those wrappers to move components around. 

But again, I think the layout looks great - I don't think you need to do anything here.